modules/bling/installers/ublue-update.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Tell build process to exit if there are any errors.
+set -oue pipefail
+
```

Comment:
may want to check if `ublue-os-update-services` is in the installed system and remove it, as it conflicts with ublue-update:
```bash
# Check if ublue-os-update-services rpm is installed, these services conflict with ublue-update
if rpm -q ublue-os-update-services > /dev/null; then
    rpm-ostree override remove ublue-os-update-services
fi
```

Reply:
[2208ea8](https://github.com/ublue-os/startingpoint/pull/135/commits/2208ea88e50bb504c440783912dd0cd56f62d008)